context/app/static/js/components/home/ExploreTools/ExploreTools.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Typography } from '@mui/material';
+
```

Comment:
Why did you choose to include this in the current PR?

Reply:
I stubbed these in to get the CSS for the homepage grid into place and make sure that the chart's positioning was all set; since there's a feature branch in between, I figured it'd be okay to keep these placeholders so there's a solid starting point when we start working on the tools section.